.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,23 +11,34 @@ Thanks for stopping by to let us know something could be better!
 Please run down the following list and make sure you've tried the usual "quick fixes":
 
   - Search the issues already opened: https://github.com/googleapis/google-api-python-client/issues
-  - If you have a question, post on Stackoverflow under the `google-api` tag.
-  - If you are reporting an issue or requesting a feature for a G Suite API, please use their [public issue tracker](https://gsuite-developers.googleblog.com/2017/03/a-new-issue-tracker-for-g-suite.html)
+  - Search the issues on our "catch-all" repository: https://github.com/googleapis/google-cloud-python
+  - Search StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform+python
```

Comment:
```suggestion
  - Search StackOverflow: https://stackoverflow.com/questions/tagged/google-cloud-platform+python
```
(nit) Let's use HTTPS by default to avoid and extra redirect to it (requesting an HTTP version results in 307). :)

Reply:
Ack, will tweak the template